### PR TITLE
update command to run ui build in build.md

### LINKED ISSUE #64: Modify `Build.md` -update endpoint command  to run build
- Update the command in readMe to run UI build server

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
update command to run UI build in build doc

Signed-off-by: Sanjeev Kumar sanjekum@redhat.com

### CODE REVIEW

File: docs/Build.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Webpack allows file specific loaders or utilities to be invoked as a part of the
...
 | `/(\.css\|.scss)$/`         | `style-loader` (dev only), `miniCssExtractPlugin.loader` (production only), `css-loader`, `sass-loader` | Handle scss/css loading/references. If dev mode, use `style-loader` for speed, else use `miniCssExtractPlugin.loader` (in combination with the `miniCssExtractPlugin` plugin) to produce/emit css file(s) |
-| `/(\.ts|tsx)$/`             | `ts-loader`                                                                                             |                                                                                                                                                               Transpiles TypeScript files into JavaScript |
+| `/(\.ts                     | tsx)\$/`                                                                                                |                                                                                                                                                                                               `ts-loader` | Transpiles TypeScript files into JavaScript |
```
Comment (matthew-chirgwin) [inline docs/Build.md]: This additional bar is breaking the table formatting. Did you add it (or did the linter)?

File: docs/Build.md [hunk 2]
```diff
@@ -204,6 +204,6 @@ The UI build is used in a `dockerfile` to produce an image that can then be depl
 - Run `npm run build`, which in turn runs a production build of the client and server
 - Clear installed dependencies, and install just production (shipped) dependencies
 - Move the built UI directory `dist` and `node_modules` to the required location
-- Sets the entrypoint to a script/command which runs the UI Server: `node dist/server/main.bundle.js`
+- Sets the entrypoint to a script/command which runs the UI Server: `node dist/server/main.js`
```
Comment (matthew-chirgwin) [inline docs/Build.md:207]: This change also needed on line 171

**Comment by pmuir** [comment]
I think you need a new line before `Signed-off-by`

**Review by matthew-chirgwin — CHANGES_REQUESTED** [review]
Thank you for this @drcoolsanjeev - just a couple of comments. I also see the DCO/Git Lint check is failing.

File: docs/Build.md (hunk 1 above)
Comment (matthew-chirgwin) [reply docs/Build.md]: Same issue seen in #66 - I am at a loss to why this is an issue (and not happened before)

Comment (drcoolsanjeev) [reply docs/Build.md]: It happened due to issue in `npm lint`

**Review by scholzj — APPROVED** [review]

File: docs/Build.md (hunk 1 above)
Comment (matthew-chirgwin) [reply docs/Build.md]: I have just seen that the `|` between ts and tsx is not escaped (like css/scss above). Hence the linter (for whatever reason now) is kicking in an treating it as a markdown column. As a part of this change, can the `\` be added?

Comment (matthew-chirgwin) [reply docs/Build.md]: For completeness, I am at a loss when I changed this file yesterday and did not have any of these issues!

**Review by matthew-chirgwin — CHANGES_REQUESTED** [review]
See comment https://github.com/strimzi/strimzi-ui/pull/65#discussion_r515056334 - which will need addressing for this file to be checked in/changed

File: docs/Build.md (hunk 1 above)
Comment (drcoolsanjeev) [reply docs/Build.md]: Added skip character (like css/scss) between ts and tsx

**Review by matthew-chirgwin — APPROVED** [review]
LGTM - thank you @drcoolsanjeev !

**Pull request merged**